Source/WebKit/UIProcess/RemoteLayerTree/RemoteAcceleratedEffectStack.mm

```diff
@@ -42,7 +43,80 @@
     [m_opacityPresentationModifier setValue:opacity];
     [m_transformPresentationModifier setValue:transform];
     [m_presentationModifierGroup flush];
+
+    if (m_filterPresentationModifierGroup) {
+        PlatformCAFilters::updatePresentationModifiersForFilters(computedValues.filter, m_filterPresentationModifiers);
+        [m_filterPresentationModifierGroup flush];
+    }
+}
+
+void RemoteAcceleratedEffectStack::initEffectsFromMainThread(PlatformLayer *layer, Seconds currentTime)
+{
+    ASSERT(!m_opacityPresentationModifier);
+    ASSERT(!m_transformPresentationModifier);
+
+    auto bounds = FloatRect(layer.bounds);
+    auto computedValues = computeValues(currentTime, bounds);
+    auto computedTransform = computedValues.computedTransformationMatrix(bounds);
+
+    auto *opacity = @(computedValues.opacity);
+    auto *transform = [NSValue valueWithCATransform3D:computedTransform];
+
+    // FIXME: Only set transform and opacity if actually present.
```

Comment:
Is this just because this would be unnecessary work, or could this result in buggy behavior?